SetupDataPkg/Tools/VariableList.py

```diff
@@ -1212,13 +1212,18 @@ def read_csv(schema, csv_path):
             read_guid = row[guid_index]
             if read_guid != '*':
                 guid = read_guid
+            #print(guid)
             knob_name = row[knob_index]
             knob_value_string = row[value_index]
-
+            #print(knob_name)

```

Comment:
can the commented out debug code get removed? it isn't required history for this PR.

Reply:
Removed